src/tests/e2e/auth/login.test.ts

```diff
@@ -0,0 +1,124 @@
+import { expect, test } from "@playwright/test";
+import {
+	deleteDefaultAccount,
+	registerThenLogoutWithDefaultAccount,
+} from "./utils";
+import { baseUrl, defaultEmail, defaultPassword } from "./constants";
+
+test.describe("Login", () => {
+	test.describe("Happy Case", () => {
+		test.beforeEach(async ({ page }) => {
+			await registerThenLogoutWithDefaultAccount(page);
+		});
+		test.afterEach(async ({ page }) => {
+			await deleteDefaultAccount(page);
+		});
+
+		test("should be able to log-in then log-out", async ({ page }) => {
+			await page.goto(`${baseUrl}/profile`);
+
+			// Login with new account
+			await page.getByLabel("Deine E-Mail Adresse").click();
+			await page.getByLabel("Deine E-Mail Adresse").fill(defaultEmail);
+			await page.getByLabel("Deine E-Mail Adresse").press("Tab");
+			await page.getByLabel("Passwort").fill(defaultPassword);
+			await page.getByLabel("Passwort").press("Enter");
+
+			await expect(page.getByText("Dein ProfilDeine Ü")).toBeVisible();
```

Comment:
`Dein ProfilDeine Ü` typo?

Reply:
This was generated by playwright codegen. I think their philosophy is to assert that a unique beginning of text (a div, title etc) being present is enough to know that the expected content is visible. 
<img width="509" alt="Bildschirmfoto 2024-03-20 um 09 24 56" src="https://github.com/technologiestiftung/giessdenkiez-de/assets/8709861/4e9d9aa7-de31-4c6a-943d-34172c1651e9">

But I agree it makes test a bit harder to read. What do you think of this suggestion to make it more readable:

```ts
const profilePage = page.getByText("Dein ProfilDeine Ü")
await expect(profilePage).toBeVisible();
```

Reply:
aah I understand, no need to change anything from my side